collections-constants.go

```diff
@@ -0,0 +1,27 @@
+package shopify
+
+const (
+	// SortOrderAlphaAsc - Alphabetically, in ascending order (A - Z).
+	SortOrderAlphaAsc = "alpha-asc"
```

Comment:
remember this also needs the type its for (So should be `CollectionSortOrderAlphaAsc`)